src/sscce.js

```diff
@@ -21,16 +21,44 @@ module.exports = async function() {
     benchmark: true,
     define: {
       timestamps: false // For less clutter in the SSCCE
```

Comment:
```suggestion
      timestamps: false, // For less clutter in the SSCCE
```

you're missing a comma here